cmd/nerdctl/main.go

```diff
@@ -175,6 +175,7 @@ func initRootCmdFlags(rootCmd *cobra.Command, tomlPath string) (*pflag.FlagSet,
 	// Experimental enable experimental feature, see in https://github.com/containerd/nerdctl/blob/main/docs/experimental.md
 	AddPersistentBoolFlag(rootCmd, "experimental", nil, nil, cfg.Experimental, "NERDCTL_EXPERIMENTAL", "Control experimental: https://github.com/containerd/nerdctl/blob/main/docs/experimental.md")
 	AddPersistentStringFlag(rootCmd, "host-gateway-ip", nil, nil, nil, aliasToBeInherited, cfg.HostGatewayIP, "NERDCTL_HOST_GATEWAY_IP", "IP address that the special 'host-gateway' string in --add-host resolves to. Defaults to the IP address of the host. It has no effect without setting --add-host")
+	AddPersistentStringSliceFlag(rootCmd, "cdi-spec-dirs", nil, nil, cfg.CDISpecDirs, "CDI_SPEC_DIRS", "The directories to search for CDI spec files. Defaults to /etc/cdi,/var/run/cdi")
```

Comment:
I'm happy to use the existing `AddPersistentStringArrayFlag` here instead, but then we need to change the command line argument to `--cdi-spec-dir` and make it clear that it's repeatable. I'm not sure whether the envvar would be split on commas then either.

Reply:
Is this env var specific to the nerdctl implementation, or is it part of the CDI specification?
I guess the env var should use `:` on Linux and `;` on Windows as the separator, just like `$PATH` (`%PATH%`)?

Reply:
The envvar is not part of the CDI specification and is a `nerdctl`-specific implementation detail. I'm happy to defer to you on how to implement this though. Should **both** the command line argument `--cdi-spec-dirs` and the `CDI_SPEC_DIRS` envvar accept a `:` (or `;`)-separated list of paths instead of a comma-separated list?

Reply:
The CLI flag should not use colon separators.

Also I wonder if the env var is really necessary. Maybe it should be revisited in a separate PR?

Reply:
OK. I can remove the envvar for now and only use the CLI flags. It is probably cleaner for the time being.